pallets/pool-registry/src/benchmarking.rs

```diff
@@ -0,0 +1,160 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+// This file is part of Centrifuge chain project.
+
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+//! Module provides benchmarking for the Pools Pallet
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
+use frame_system::RawOrigin;
+use sp_std::vec;
+
+use super::*;
+
+const POOL: u64 = 0;
+
+benchmarks! {
+	where_clause {
+	where
+		T: Config<PoolId = u64>,
+	}
+
+	// create {
```

Comment:
It's dirty to leave commented code in there, but should we keep it as a reference for the follow up PR to implement benchmarking?